Gemfile

```diff
@@ -1,5 +1,4 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
-gem 'pdk', git: 'https://github.com/puppetlabs/pdk.git', branch: 'master'
```

Comment:
Probably want to mention why the Gemfile is being changed in a PR about Travis.

Reply:
this file was changed by pdk_update

Reply:
you can check Gemfile for [facter_task](https://github.com/puppetlabs/puppetlabs-facter_task/blob/db743cdf2746d870bfc35403f656cd1b54bee5da/Gemfile#L2)